KsApi/extensions/ApolloClient+RAC.swift

```diff
@@ -27,6 +27,7 @@ extension ApolloClient {
           observer.send(value: data)
           observer.sendCompleted()
         case let .failure(error):
+          print("🔴 [KsApi] ApolloClient query failure - error : \((error as NSError).description)")
```

Comment:
line 21 and 24 handle the error returned from the API. This failure case is run when there's a network failure or something similar.